versioned_docs/version-24.3.0/private-networks/how-to/configure/consensus/qbft.md

```diff
@@ -183,6 +183,12 @@ The properties with specific values in the QBFT genesis files are:
 
 To start a node on a QBFT private network, use the [`--genesis-file`](../../../../public-networks/reference/cli/options.md#genesis-file) option to specify the custom genesis file.
 
+:::note Sync nodes for BFT
+
+If you're using a Byzantine Fault Tolerance (BFT) consensus mechanism, such as QBFT, set your node's [synchronization mode](/public-networks/get-started/connect/sync-node#run-a-full-node) to either `FAST` or `FULL`. Configure [`-- sync-mode`](/public-networks/get-started/connect/sync-node#run-a-full-node) to ensure the network is stable and to prevent block creation issues.
```

Comment:
the second sentence feels redundant to me, it's not adding anything - I'd leave it out

Reply:
Thanks @macfarla . I will switch to the `development` docs.